docker/defaults.yaml

```diff
@@ -2,32 +2,33 @@
 
 docker:
   process_signature: '/usr/bin/docker'
-  install_pypi_pip: False
   install_docker_py: True
   python_package: docker
   refresh_repo: True
   configfile: /etc/default/docker
   config: []
   daemon_config: {}
+  version:
```

Comment:
This will evaluate to a literal string: `'None'` -- as discussed [here](https://github.com/saltstack-formulas/postgres-formula/pull/217).  Even if that's intended, better to be explicit.

Reply:
Set "None" vars to '' explicitly.